modules/shortcodes/wordads.php

```diff
@@ -53,14 +57,41 @@ static function wordads_shortcode_html( $atts, $content = '' ) {
 			return '<div>' . __( 'The WordAds module is not active', 'jetpack' ) . '</div>';
 		}
 
-		$html = '<div class="jetpack-wordad" itemscope itemtype="https://schema.org/WPAdBlock">';
-
-		$html .= '</div>';
-
+		$html = '<div class="jetpack-wordad" itemscope itemtype="https://schema.org/WPAdBlock"></div>';
 		$html = $wordads->insert_inline_ad( $html );
 
 		return $html;
 	}
+
+	public static function gutenblock_render( $attr ) {
+		global $wordads;
+
+		/** This filter is already documented in modules/wordads/wordads.php `insert_ad()` */
+		if ( is_feed() || apply_filters( 'wordads_inpost_disable', false ) ) {
+			return '';
+		}
+
+		if ( $wordads->option( 'wordads_house' ) ) {
+			return $wordads->get_ad( 'inline', 'house' );
+		}
+
+		$section_id = 0 === $wordads->params->blog_id ? WORDADS_API_TEST_ID : $wordads->params->blog_id . '6'; // 6 is to keep track of gutenblock ads
+		$align = 'center';
+		if ( isset( $attr['align'] ) && in_array( $attr['align'], array( 'left', 'center', 'right' ) ) ) {
+			$align = $attr['align'];
+		}
+		$align = 'align' . $align;
+
+		$format = 'mrec';
+		if ( isset( $attr['format'] ) && in_array( $attr['format'], array_keys( WordAds::$ad_tag_ids ) ) ) {
+			$format = $attr['format'];
+		}
+
+		$height = WordAds::$ad_tag_ids[$format]['height'];
```

Comment:
Lint says variable array keys should be surrounded by space
```suggestion
		$height = WordAds::$ad_tag_ids[ $format ]['height'];
```

Reply:
Updated 👍